src/pybind/mgr/dashboard_v2/__init__.py

```diff
@@ -25,3 +25,5 @@ def __getattr__(self, item):
     logging.basicConfig(level=logging.DEBUG)
     logger = logging.getLogger(__name__)
     logging.root.handlers[0].setLevel(logging.DEBUG)
+    os.environ['PATH'] = '{}:{}'.format(os.path.abspath('../../../../build/bin'),
```

Comment:
Could you add this change to a single commit?

Reply:
I agree. This seems a different logical change from the rest of the patch.